e2e/cypress/integration/api/iam/iam_integration.spec.ts

```diff
@@ -214,6 +231,22 @@ describeIfIAMV2p1('assigning projects', () => {
       });
     });
 
+    it(`when creating ${iamResources} without any projects ` +
```

Comment:
`when creating ${iamResources} without any projects and project assignment permission is missing for (unassigned), fails to create unassigned ${iamResource}`